coil/src/main/java/dev/chrisbanes/accompanist/coil/Coil.kt

```diff
@@ -224,6 +224,11 @@ fun CoilImage(
     }
 }
 
+/**
+ * Value for a [IntSize] dimension, where the dimension is not specified or is unknown.s
```

Comment:
Thanks for making the update!

Minor typo in the javadoc `.s` at the end.

Reply:
Woops, thanks!